src/View/Game/game.html.twig

```diff
@@ -0,0 +1,17 @@
+{% extends 'layout.html.twig' %}
+
+    {% block title %}
+        Stercket Battle
+    {% endblock %}
+    {% block link %}
+            <link rel="stylesheet" href="/assets/css/game.css">
+            <link href="/assets/fonts/PasseroOne-Regular.ttf">
```

Comment:
Isn't the font use on all the pages ?
So it'll probably be inserted on the layout.